arch/src/aarch64/fdt.rs

```diff
@@ -838,3 +838,18 @@ fn print_node(node: fdt_parser::node::FdtNode<'_, '_>, n_spaces: usize) {
         print_node(child, n_spaces + 2);
     }
 }
+
+pub fn get_gic_dist_redist(dtb: &mut File) -> (u64, u64, u64, u64) {
+    let blob = fdt_file_to_vec(dtb).unwrap();
+    let fdt = fdt_parser::Fdt::new(&blob).unwrap();
+    let node = fdt.find_compatible(&["arm,gic-v3"]).unwrap();
```

Comment:
Instead of unwrap should we handle the failure also?

Reply:
I don't think we can handle the failure - we need this node to be present in the host fdt.